src/index.html

```diff
@@ -15,13 +22,330 @@
+      <nav class="nav">
+        <ul class="nav__menu">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__menu-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__menu-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__menu-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__menu-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__menu-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__menu-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__menu-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__menu-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="cards">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="  APPLE A1419 iMac"
+          />
+          <h2 class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__cost">
+            <p class="card__price">Price:</p>
+            <p class="card__total">$2,199</p>
+          </div>
+
+          <button
+            data-qa="card-hover"
+            class="card__button"
+          >
+            Buy
+          </button>
+        </div>
+        <div class="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="  APPLE A1419 iMac"
```

Comment:
There is an extra space at the beginning of the `alt` attribute value. Consider removing it to maintain consistency and avoid potential issues with screen readers.